### PR TITLE
Standardize wdl description

### PR DESCRIPTION
**Pull request by bshifaw**
Edited ./pipeline WDLs such that the workflow description follow the following format:

```
WorkflowName {

    meta {
        description: "Workflow description"
    }
    parameter_meta {
        input: "Input description"
    }

    input{
    ...
    }
...
}
```

### CODE REVIEW

File: pipelines/ONT/Epigenomics/ONTMethylation.wdl [hunk 1]
```diff
@@ -7,6 +7,20 @@ import "../../../tasks/Preprocessing/Guppy.wdl" as Guppy
 import "../../../tasks/Utility/Finalize.wdl" as FF
 
 workflow ONTMethylation {
+
+    meta {
```
Comment (SHuang-Broad) [inline pipelines/ONT/Epigenomics/ONTMethylation.wdl:11]: Thank you!

File: pipelines/ONT/VariantCalling/ONTWholeGenome.wdl [hunk 2]
```diff
@@ -14,6 +8,31 @@ import "../../../tasks/Utility/Finalize.wdl" as FF
 import "../../../tasks/QC/SampleLevelAlignedMetrics.wdl" as COV
 
 workflow ONTWholeGenome {
+
+    meta {
+        description: "A workflow that performs single sample variant calling on Oxford Nanopore reads from one or more flow cells. The workflow merges multiple samples into a single BAM prior to variant calling."
```
Comment (SHuang-Broad) [inline pipelines/ONT/VariantCalling/ONTWholeGenome.wdl:13]: Typo: not `multiple samples`, but `multiple flowcells`

File: pipelines/PacBio/Alignment/PBFlowcell.wdl [hunk 3]
```diff
@@ -20,6 +13,30 @@ import "../../../tasks/Transcriptomics/MASSeq.wdl" as MAS
 import "../../../tasks/Utility/JupyterNotebooks.wdl" as JUPYTER
 
 workflow PBFlowcell {
+
+    meta {
```
Comment (SHuang-Broad) [inline pipelines/PacBio/Alignment/PBFlowcell.wdl:17]: @jonn-smith can you please add your description of the MASseq part?

Comment (SHuang-Broad) [reply pipelines/PacBio/Alignment/PBFlowcell.wdl:17]: For the genome side, here's my proposed description:

```
The workflow performs the alignment of an SMRT cell's worth of data to a reference.
For genomic sequencing data, the workflow also optionally performs CCS correction if the data is from a CCS library but did not get corrected on-instrument.
```

File: pipelines/PacBio/Assembly/PBAssembleWithHifiasm.wdl [hunk 4]
```diff
@@ -1,28 +1,16 @@
...
-    input {
-        Array[File] ccs_fqs
-
-        File ref_map_file
-        String participant_name
-        String prefix
 
-        String gcs_out_root_dir
+    meta {
+        description: "A workflow that performs single sample genome assembly on PacBio HiFi reads from one or more flow cells. The workflow merges multiple samples into a single BAM prior to genome assembly and variant calling."
```
[the remark this answers is not in the document]
Comment (SHuang-Broad) [reply pipelines/PacBio/Assembly/PBAssembleWithHifiasm.wdl:12]: Also, 

```
The multiple SMRT cells data are merged prior to assembly.
```

File: pipelines/PacBio/VariantCalling/PBCCSWholeGenome.wdl [hunk 5]
```diff
@@ -14,6 +8,31 @@ import "../../../tasks/Utility/Finalize.wdl" as FF
 import "../../../tasks/QC/SampleLevelAlignedMetrics.wdl" as COV
 
 workflow PBCCSWholeGenome {
+
+    meta {
+        description: "A workflow that performs single sample variant calling on PacBio HiFi reads from one or more flow cells. The workflow merges multiple samples into a single BAM prior to variant calling."
```
Comment (SHuang-Broad) [inline pipelines/PacBio/VariantCalling/PBCCSWholeGenome.wdl:13]: Again, not multiple samples but multiple SMRT cells.

File: pipelines/PacBio/VariantCalling/PBCLRWholeGenome.wdl [hunk 6]
```diff
@@ -14,6 +8,25 @@ import "../../../tasks/Utility/Finalize.wdl" as FF
 import "../../../tasks/QC/SampleLevelAlignedMetrics.wdl" as COV
 
 workflow PBCLRWholeGenome {
+
+    meta {
+        description: "A workflow that performs single sample variant calling on PacBio CLR reads from one or more flow cells. The workflow merges multiple samples into a single BAM prior to variant calling."
```
Comment (SHuang-Broad) [inline pipelines/PacBio/VariantCalling/PBCLRWholeGenome.wdl:13]: ditto.

File: pipelines/TechAgnostic/Utility/LRConvertBCF.wdl [hunk 7]
```diff
@@ -1,15 +1,20 @@
...
-## transient issues stemming from the LRJointCallGVCFs workflow. Should be removed eventually.
-############################################################################################
-
 import "../../../tasks/VariantCalling/GLNexus.wdl" as GLNexus
 import "../../../tasks/Utility/Hail.wdl" as Hail
 import "../../../tasks/Utility/Finalize.wdl" as FF
 
 workflow LRConvertBCF {
+
+    meta {
```
Comment (SHuang-Broad) [inline pipelines/TechAgnostic/Utility/LRConvertBCF.wdl:9]: we need to create a ticket for this, given the comment there.

**Review by SHuang-Broad — CHANGES_REQUESTED** [review]
Some minor changes, then good to go.

File: pipelines/PacBio/Assembly/PBAssembleWithHifiasm.wdl (hunk 4 above)
[the remark this answers is not in the document]
Comment (bshifaw) [reply pipelines/PacBio/Assembly/PBAssembleWithHifiasm.wdl:12]: @SHuang-Broad should this be all wdl descriptions or just the ones you pointed out?

File: pipelines/TechAgnostic/Utility/LRConvertBCF.wdl (hunk 7 above)
Comment (bshifaw) [reply pipelines/TechAgnostic/Utility/LRConvertBCF.wdl:9]: #395

File: pipelines/PacBio/VariantCalling/PBCLRWholeGenome.wdl (hunk 6 above)
Comment (bshifaw) [reply pipelines/PacBio/VariantCalling/PBCLRWholeGenome.wdl:13]: done

File: pipelines/PacBio/VariantCalling/PBCCSWholeGenome.wdl (hunk 5 above)
Comment (bshifaw) [reply pipelines/PacBio/VariantCalling/PBCCSWholeGenome.wdl]: done

File: pipelines/ONT/VariantCalling/ONTWholeGenome.wdl (hunk 2 above)
Comment (SHuang-Broad) [reply pipelines/ONT/VariantCalling/ONTWholeGenome.wdl]: All. I don't think we have multi-sample workflows except two or three for joint calling.

File: pipelines/PacBio/Assembly/PBAssembleWithHifiasm.wdl (hunk 4 above)
[the remark this answers is not in the document]
Comment (SHuang-Broad) [reply pipelines/PacBio/Assembly/PBAssembleWithHifiasm.wdl]: For this assembly, just this once.

**Pull request merged**